src/types/targetValidators.ts

```diff
@@ -325,6 +325,10 @@ export const validateStreamConfig = (
     )
   }
 
+  if (streamConfig.streamLogo && typeof streamConfig.streamLogo !== 'string') {
+    streamConfig.streamLogo = undefined
```

Comment:
A default (eg "logo.png") would be nice, as it would be one less thing to configure (and can mean more concise config objects if the standard is followed)